frontend/components/ConfirmDataCollectionDisableModal/_styles.scss

```
@@ -1,12 +1,25 @@
+  p {
+    margin: 0;
+  }
+
+  &__dataset-block {
+    @include flex-column-8px-gap;
+  }
+
   &__dataset-list {
-    margin: 0 0 $pad-medium 0;
+    @include flex-column-4px-gap;
+    margin: 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)